django_spanner/base.py

```diff
@@ -113,7 +117,7 @@ def _nodb_connection(self):
 
     def get_connection_params(self):
         return {
-            "project": self.settings_dict["PROJECT"],
+            "project": os.environ["GOOGLE_CLOUD_PROJECT"],
```

Comment:
Few fixes to run the tests right on the emulator

Reply:
Hm-m, this requires a careful look. Some parts of piping client to emulator code uses env variables. I suppose setting both evn vars and the same Django settings is not something we should do.